packages/@aws-cdk/custom-resources/test/aws-custom-resource/aws-custom-resource.test.ts

```diff
@@ -84,7 +84,7 @@ test('onCreate defaults to onUpdate', () => {
         Key: 'my-key',
         Body: 'my-body'
       },
-      physicalResourceIdPath: 'ETag'
+      physicalResourceId: PhysicalResourceId.fromResponsePath('ETag')
```

Comment:
The expectations (stack output) in this file have changed since `physicalResourceId` is now an object with either `literalId` or `responsePath`.

Reply:
Done